### PR TITLE
testStart and testStop methods should only be used once check

### PR DESCRIPTION
**Pull request by VarS311**
PR for issue #44

### CODE REVIEW

File: apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/StartAndStopCheckTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (JimmyRojas) [inline apex-checks/src/test/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/StartAndStopCheckTest.java:1]: invalid class header

File: apex-checks/src/test/resources/checks/startAndStopClass.cls [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            System.assert(x.equals(y));
+            test.stopTest();
+	}
+
+        //also valid, using none
+        public static testMethod String aTestMethod() {
+            System.assert(x.equals(y));
+	}
+}
+
```
Comment (JimmyRojas) [inline apex-checks/src/test/resources/checks/startAndStopClass.cls:43]: remove blank line

**Comment by JimmyRojas** [comment]
just 2 minor updates

File: apex-checks/src/test/resources/checks/startAndStopClass.cls (hunk 2 above)
Comment (VarS311) [inline apex-checks/src/test/resources/checks/startAndStopClass.cls:43]: You mean the commented line? @JimmyRojas

File: apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/StartAndStopCheck.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+     */
+    private final String START = "STARTTEST";
+
+    /**
+     * One of the Keywords for the second part of the expression.
+     */
+    private final String STOP = "STOPTEST";
+    
+    /**
+     *
```
Comment (JimmyRojas) [inline apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/StartAndStopCheck.java]: missing doc

File: apex-checks/src/test/resources/checks/startAndStopClass.cls (hunk 2 above)
Comment (JimmyRojas) [inline apex-checks/src/test/resources/checks/startAndStopClass.cls:43]: it's ok

File: apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/StartAndStopCheck.java (hunk 3 above)
Comment (VarS311) [inline apex-checks/src/main/java/org/fundacionjala/enforce/sonarqube/apex/checks/testrelated/StartAndStopCheck.java]: Added them in the last commit :+1:

**Pull request merged**